src/metabase/driver/postgres.clj

```diff
@@ -163,7 +163,13 @@
     driver.common/advanced-options-start
     (assoc driver.common/additional-options
            :placeholder "prepareThreshold=0")
-    driver.common/default-advanced-options]
+    driver.common/default-advanced-options
+    {:name         "database-enable-actions"
+     :type         :boolean
+     :default      false
+     :display-name (deferred-tru "Enable experimental actions")
```

Comment:
`database-enable-actions` isn't something that goes in Database `details`... it goes in Database `settings`. So you don't need to add this.

Reply:
I need to implement https://github.com/metabase/metabase/issues/19409 really quick so you can edit the Database-local Setting value

Reply:
I put the persistence enabled information into the options ([pg driver](https://github.com/metabase/metabase/blob/master/src/metabase/driver/postgres.clj#L54-L56)) but i suppose settings makes more sense. We should switch that over.